src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -22,6 +22,7 @@ import { useSynapseContext } from '../../utils/SynapseContext'
 import { AddToDownloadListRequest } from '../../utils/synapseTypes/DownloadListV2/AddToDownloadListRequest'
 import { useGetEntityChildren } from '../../utils/hooks/SynapseAPI/useGetEntityChildren'
 import useGetQueryResultBundle from '../../utils/hooks/SynapseAPI/useGetQueryResultBundle'
+import { useGetDownloadListStatistics } from '../../utils/hooks/SynapseAPI/useGetDownloadListStatistics'
```

Comment:
Refetch the statistics after adding items to the cart (this will update other UI on the site, like the download notification count in the header)